### ISSUE TITLE
OpenAL error spam in stdout

### ISSUE THREAD

**Issue by illwieckz** [DaemonEngine/Daemon#445]
With 0.52 RC2 on Linux my stdout is spammed with many of these lines (no variation):

```
AL lib: (WW) alSetError: Error generated on context 0x56041150d8a0, code 0xa003
AL lib: (WW) alSetError: Error generated on context 0x56041150d8a0, code 0xa003
AL lib: (WW) alSetError: Error generated on context 0x56041150d8a0, code 0xa003
```

This does not seems to be part of the game console logging system (not seen in game, and `logs.suppression.enabled` has no effect on it).

It seems to happen on sound events.

I didn't noticed sound rendering errors though.

**Comment by illwieckz** [comment]
This spam may induce strong performance issues for people running the game on a terminal.

**Comment by illwieckz** [comment]
Here someone reported a similar issue with another software: https://github.com/panda3d/panda3d/issues/180
And that other software implemented that fix: https://github.com/panda3d/panda3d/commit/5980b0b9abb77c7a2a5b4709de0255fb6db03a48

**Comment by illwieckz** [comment]
```
]/audio.al.checkAllCalls 1
Warn: Unhandled OpenAL error on line 542: Invalid value

]/devmap plat23
[…]

]AL lib: (WW) alSetError: Error generated on context 0x5605e93947e0, code 0xa004
Warn: Unhandled OpenAL error on line 128: Invalid operation 

]AL lib: (WW) alSetError: Error generated on context 0x5605e93947e0, code 0xa003
Warn: Unhandled OpenAL error on line 721: Invalid value 
```

It looks like it's all about the [`src/engine/audio/ALObjects.cpp`] file.

**Comment by illwieckz** [comment]
I get this with the release build produced with the docker build, not when I build it myself with classic cmake calls.

**Comment by illwieckz** [comment]
The log spam is assumed to be fixed by #455. #456 is believed to be there for fixing the underlying warnings we silence.
